en/identity-server/7.1.0/docs/apis/restapis/configs.yaml

```diff
@@ -1208,10 +1241,21 @@ components:
         displayName:
           type: string
           example: basic
+        description:
+          type: string
+          example: "Description for user defined local authenticator configuration."
```

Comment:
Isn't this model commonly used for both user defined and system defined ? I don't think we can give this description in that case